bolt-sidecar/src/state/execution.rs

```diff
@@ -801,9 +812,11 @@ mod tests {
         let anvil = launch_anvil();
         let client = StateClient::new(anvil.endpoint_url());
 
-        let max_comms = NonZero::new(10).unwrap();
-        let max_gas = NonZero::new(10_000_000).unwrap();
-        let mut state = ExecutionState::new(client.clone(), max_comms, max_gas).await?;
+        let limits: Limits = Limits {
+            max_commitments_per_slot: NonZero::new(10).unwrap(),
+            max_committed_gas_per_slot: NonZero::new(10_000_000).unwrap(),
+        };
+        let mut state = ExecutionState::new(client.clone(), limits).await?;
```

Comment:
you can also do `ExecutionState::new(client.clone(), Limits::default)` if you want - it's ok like this tho!